source/geometries/FullRingInfinity.cc

```diff
@@ -422,6 +464,117 @@ void FullRingInfinity::BuildSensors()
   }
 }
 
+void FullRingInfinity::BuildWires()
+{
+  // Add simple detector for charge
+  G4Box* chdet_solid = new G4Box("WIRE", wire_pitch_/2.,
+                                 axial_length_/2., chdet_thickn_/2);
+  G4LogicalVolume* chdet_logic =
+    new G4LogicalVolume(chdet_solid, LXe_, "WIRE");
```

Comment:
These volume are not real one: here I'm just assuming that the charge is detected by something, with efficiency close to 100%, and this thing does not affect the propagation of optical photons at first order. In a real experiment, the wire will be very thin, not a bar like these volumes. It's a simplification, since the only thing we're interested into now is to calculate how many electrons we detect and how they are distributed among the different wires.